src/Mutator/Operator/Finally_.php

```diff
@@ -95,6 +95,6 @@ private function hasAtLeastOneCatchBlock(Node $node): bool
         $parentNode = ParentConnector::getParent($node);
         Assert::isInstanceOf($parentNode, Node\Stmt\TryCatch::class);
 
-        return count($parentNode->catches) > 0;
+        return $parentNode->catches !== [];
```

Comment:
should this really be changed? I don't feel strongly about using this form rather than `count()` but IMO should be done on the whole codebase in a dedicated PR rather than here